### PR TITLE
[gardening] Remove REQUIRE-lines from fixed crashers, fix invalid Swift URLs, …

### PR DESCRIPTION
**Pull request by practicalswift**
Areas of gardening:
* Remove `REQUIRE` lines from fixed crashers.
* Fix invalid Swift URLs in headers.
* PEP-8: Stay within 79 chars.
* Use consistent headers.
* Fix recently introduced typos.

### CODE REVIEW

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

File: stdlib/public/SDK/Foundation/Data.swift [hunk 1]
```diff
@@ -103,7 +103,7 @@ public final class _DataStorage {
     public var _length: Int
     public var _capacity: Int
     public var _needToZero: Bool
-    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)? = nil
+    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)?
```
Comment (jrose-apple) [inline stdlib/public/SDK/Foundation/Data.swift]: Like the ones in the standard library, I'd hesitate about changing these without asking someone from Foundation.

Comment (practicalswift) [inline stdlib/public/SDK/Foundation/Data.swift]: @jrose-apple Whoops, my bad. I've now updated my gardening scripts not to hunt for redundant initializations to `nil` :-)  Thanks for notifying.

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

File: stdlib/public/SDK/Foundation/Data.swift (hunk 1 above)
Comment (practicalswift) [inline stdlib/public/SDK/Foundation/Data.swift]: @jrose-apple I've now removed this specific commit from the PR.

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
The Linux buildbot machine that ran the last Linux test appears to have no space left on device.

**Comment by jrose-apple** [comment]
cc @najacque

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Pull request merged**